sopel_modules/github/webhook.py

```diff
@@ -156,12 +156,14 @@ def handle_auth_response():
     channel = state.split(':')[1]
 
     data = {'client_id': sopel_instance.config.github.client_id,
-             'client_secret': sopel_instance.config.github.secret,
+             'client_secret': sopel_instance.config.github.client_secret,
              'code': code}
     raw = requests.post('https://github.com/login/oauth/access_token', data=data, headers={'Accept': 'application/json'})
     try:
         res = json.loads(raw.text)
 
+        if 'error' in res:
+            raise ValueError('{err}: {des}'.format(err=res['error'], des=res['error_description'])
```

Comment:
```suggestion
            raise ValueError('{err}: {desc}'.format(err=res['error'], desc=res['error_description'])
```

This is just slightly clearer for future maintainers. It took me a moment to figure out what "des" meant. 😅

Reply:
You're right!